src/validators/ClientSideJsonValidator.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace hrzg\widget\validators;
+
+use yii\validators\Validator;
+
+class ClientSideJsonValidator extends Validator
+{
+    public function validateValue($value)
+    {
+        return [];
+    }
+
+    public function clientValidateAttribute($model, $attribute, $view)
+    {
+        return <<<JS
```

Comment:
Does it work, when using multiple JSONEditor on the same Page?

Reply:
The ClientSideJsonValidator works with the first json-editor it finds. Currently, Widgets2 uses only one json-editor at a time.

 The publication module relies on Widgets2 as a dependency, I manually tested it and found no errors. It's worth noting that the publication module doesn't actively use the ClientSideJsonValidator. Instead, relies on the the  regular json-editor error messages during interaction.

If you have any potential scenarios that could conflict with this pull request, please let me know. I know this PR could create more issues than it solves :)

Reply:
I added an extra check for the editor. In any case, if for some reason an error occurs during this client validation step, The page will just submit the form normally (no AJAX) and proceed with the normal flow, meaning backend validation and showing an error summary

Reply:
Is `ClientSideJsonValidator` specific to widgets or would it actually belong to https://github.com/dmstr/yii2-json-editor ?